src/libs/ui/src/molecules/UserInfoReview/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Button } from '@ui/atoms/Button';
+
+interface UserInfoProps {
+  name: string;
+  location: string;
+  ageGroup: string;
+  reviewCount: number;
+  votesCount: number;
+}
+
+const UserInfoReview: React.FC<UserInfoProps> = ({ name, location, ageGroup, reviewCount, votesCount }) => (
+  <div className='inline-block max-w-[22%] min-w-[22%] pt-2.5 mt-0 !font-SansSerif'>
+    <div className='mb-[10px] '>
+      <Button className='hidden lg:block font-bold bg-transparent'>
+        <h3>{name}</h3>
+      </Button>
+    </div>
+    <div className='hidden lg:block '>
+      <div className='mt-[5px] mb-[10px] mr-[5px] '>
+        <span>{location}</span>
+      </div>
+      <div className='mt-[5px] mb-[10px] block'>
+        <div>
+          <span className='text-[16px] mr-[5px]'>Review</span>
```

Comment:
create Class for **text-[16px] mr-[5px**] and **mr-[5px] font-bold**, and reuse wherever you required